content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -17,7 +18,7 @@ const ActivityFeedItem = React.createClass({
           <a className="feed-link" href={site.url} ref="link">{prettyUrl(site.url)}</a>
         </div>
         <div className="feed-stats">
-          <div>1:26pm</div>
+          <div>{site.lastVisitDate && moment(site.lastVisitDate / 1000).format("h:mma")}</div>
```

Comment:
it's probably just our fake data that's missing it... I was just lazy :laughing: